### DISCUSSION TITLE
Carbon Neutrality (Project 242)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#77] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 242](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/blob/main/projects/Carbon%20Neutrality)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by kbdankhara** [comment]
Hi, I am a student at IISc and interested in working and collaborating on this project

**Comment by hrcheung** [comment]
Coming in from Northeastern University. Studying this database for coursework and potential research opportunity!

**Comment by AranaaOO** [comment]
Hello, I am a student at Kwame Nkrumah University of Science and Technology and I am very excited to work on this project.

**Comment by thedrayakintola** [comment]
Hi, I am a student at West Virginia University, and I am excited to start work on this project. I am open for discussions on this project!

**Comment by latent-variable-17** [comment]
Hi, I am a student at Amrita Vishwa Vidyapeetham , and I am excited to start work on this project. I am open for discussions on this project!

**Comment by thedrayakintola** [comment]
Hi @ all, I need clarity on the specific dataset to use out of all the given csv/excel files (GCB2021v34_MtCO2_flat, GCB2021v34_percapita_flat, and GCB2021v34_sources_flat). Thank you!

**Reply by ydong9107** [reply]
You can pick up one related one column to use. It will be better if you used more columns to make a better model

**Reply by thedrayakintola** [reply]
Thanks for the response, do you mean that I can use any of these ((GCB2021v34_MtCO2_flat, GCB2021v34_percapita_flat, and GCB2021v34_sources_flat)?

**Reply by thedrayakintola** [reply]
Let me make my question clearer. Out of these three CSV files **(GCB2021v34_MtCO2_flat, GCB2021v34_percapita_flat, and GCB2021v34_sources_flat)** from the link to the dataset, which are we to use for the project. Or, can we choose any from these three datasets?

**Reply by ydong9107** [reply]
You can choose any of these for modeling.

**Reply by thedrayakintola** [reply]
Thank you for the assistance!

**Comment by AkankshaCherala** [comment]
Hii I'm at VNRVJIET ,and im very exicted to work in the project. And I'm open for the discussion. Where we can come out with different ideas. Lets grab this opportunity and try our best.
